source/_includes/common-tasks/network_storage.md

```diff
@@ -0,0 +1,81 @@
+## Network storage
+
+You can configure both Network File Share (NFS) and Windows samba (CIFS) targets to be used within Home Assistant and add-ons.
+To list all your currently connected network storages go to **{% my storage title="Settings > System > Storage" %}** in the UI.
+
+{% if page.installation == "os" %}
+
+<div class='note'>
+  You need to update to Home Assistant Operating System 10.2 before you can use this feature.
+</div>
+
+{% endif %}
+
+<p class='img'>
+  <picture>
+    <source srcset="/images/screenshots/network-storage/list_dark.png" media="(prefers-color-scheme: dark)">
+    <img src="/images/screenshots/network-storage/list_light.png">
+  </picture>
+  Screenshot of the list of network shares inside the storage panel.
+</p>
+
+### Add a new new network storage
+
+1. Go to **{% my storage title="Settings > System > Storage" %}** in the UI.
+1. Click the "Add network storage" button.
+1. Fill out all the information for your network storage
+1. Press "Connect".
+
+<p class='img'>
+  <picture>
+    <source srcset="/images/screenshots/network-storage/connect_dark.png" media="(prefers-color-scheme: dark)">
+    <img src="/images/screenshots/network-storage/connect_light.png">
+  </picture>
+  Screenshot of connecting a new network storage.
+</p>
+
+#### Network storage configuration
+
+{% configuration_basic "hassio.network_share" %}
+Name:
+  description: This is the name that will be used for the mounted directory on your system.
+Usage:
+  description: Here you select how the target should be used.
```

Comment:
```suggestion
  description: Here, you select how the target should be used.
```
Why do I have to specify if it is used for backup or media? What's the difference? Can I change my mind later?